README.md

```diff
@@ -0,0 +1,44 @@
+skitter_api is a small wrapper around a set of [FlareSolverr](https://github.com/FlareSolverr/FlareSolverr) instances for load
+balancing and redundancy purposes. Essentially a series of docker containers
+comprised of gluetun providing a VPN connection and FlareSolverr providing CF
+workarounds are kept alive and proxied to.
+
+Currently these workers choose from a shared queue table stored in postgres;
+this is for ease-of-adapting from a previous project that was trivially
+shardable. This is far from ideal for a number of reasons. There's an initial
+effort to move to rabbitmq queues instead.
+
+Ensure you create a valid `etc/nginx/auth/skitter.htpasswd` file.
+
+`/etc/hosts` on the deploy target should contain an entry for polaris.fanfic.dev
+
+
+Create the docker container `docker/config.tsv`; see `docker/config.ex.tsv` for an
+example. The general format is:
+	`user_secret_name,pass_secret_name,subdomain,region`
+
+Create the secret files referenced by the config, for example:
+```sh
```

Comment:
These aren't commands, they're file names. How about turning them into a list of backtick wrapped entries like you've wrapped the other file names in this README. Would removing the `./` prefix on these make it clearer?

Reply:
Something like:
 - `user_secret_name.csv`
 - `pass_secret_name.csv`
 - `subdomain.csv`
 -  `region.csv`

Reply:
Done